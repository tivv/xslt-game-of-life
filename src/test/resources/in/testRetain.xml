<html>
    <body>
        <table>
            <tr><td class='live'/><td class='live'/></tr>
            <tr><td class='live'/><td class='live'/></tr>
        </table>
    </body>
</html>