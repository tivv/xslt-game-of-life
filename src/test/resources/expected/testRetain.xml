<html>
<body>
<table>
<tr>
<td class="live"></td><td class="live"></td>
</tr>
<tr>
<td class="live"></td><td class="live"></td>
</tr>
</table>
</body>
</html>
